inputs-optional.tf

```diff
@@ -84,26 +84,20 @@ variable "enable_read_only_group_access" {
   description = "Enables read only group support for accessing the database"
 }
 
-variable "add_multiple_admin_groups" {
-  type        = bool
-  default     = false
-  description = "Allows to add multipe AAD admin group for the postgres DB"
+variable "collation" {
+  type        = string
+  default     = "en_GB.utf8"
+  description = "Specifies the Collation for the Azure PostgreSQL Flexible Server Database, which needs to be a valid PostgreSQL Collation."
 }
 
-variable "additional_admin_groups" {
+variable "charset" {
   type        = string
-  default     = null
-  description = "Comma seperated additional admin groups"
+  default     = "utf8"
+  description = "Specifies the Charset for the Azure PostgreSQL Flexible Server Database, which needs to be a valid PostgreSQL Charset."
 }
 
-variable "add_multiple_readonly_groups" {
+variable "high_availability" {
   type        = bool
   default     = false
-  description = "Allows to add multipe AAD read only group for the postgres DB"
+  description = "Enable high availability mode for the PostgreSQL Flexible Server."
```

Comment:
I think this should have a readme update should make it clear this needs to be enabled in prod and probably aat/staging and perftest/test

Reply:
```suggestion
  description = "Overrides the automatic selection of high availability mode for the PostgreSQL Flexible Server. Generally you shouldn't set this yourself."
```